private/getParProfilesByOptimization.m

```diff
@@ -68,44 +68,47 @@
 % * 2016/10/04 Daniel Weindl
 % * 2016/10/12 Paul Stapor
 
+    %% No check of inputs (except figure), already done in getParameterProfiles
 
-
-%% No check of inputs (except figure), already done in getParameterProfiles
-
-% Depending on parallel computation mode, the figure will be updated
-if (nargin >= 4)
-    fh = varargin{1};
-    options.fh = fh;
-else
-    fh = [];
-end
-
-%% Profile calculation
-if strcmp(options.comp_type,'sequential')
-    for iPar = options.profile_optim_index
-        % Define the negative log-posterior function
-        % (fmincon needs the neagtive log posterior for optimization)
-        % with the profile parameter fixed
-        parameters = optimizeProfileForParameterI(parameters, objective_function, iPar, options, fh);
+    % Depending on parallel computation mode, the figure will be updated
+    if (nargin >= 4)
+        fh = varargin{1};
+        options.fh = fh;
+    else
+        fh = [];
     end
     
-elseif strcmp(options.comp_type,'parallel')
-    parfor iPar = options.profile_optim_index
-        % Define the negative log-posterior function
-        % (fmincon needs the neagtive log posterior for optimization)
-        % with the profile parameter fixed
-        optimizeProfileForParameterI(parameters, objective_function, iPar, options, fh);
+    %% Check for fixed parameters
```

Comment:
whats the issue here?

Reply:
The issue was that the current implementation is indeed not working for profiles...